apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/UrlSignerV4TestData.json

```diff
@@ -1,136 +1,149 @@
 ﻿[
-    {
```

Comment:
At this stage, what are the actual semantic changes in the JSON file? I'd prefer not to have these whitespace changes if there's no purpose to them. I'm happy to have a canonicalized format for the JSON files when we get to a conformance repo, potentially after some discussion about that formatting (I'm not sure why there's a two space indent for "object with array" but a four space indent for "field within object" here) - but let's work out the semantics first.